gcc/rust/checks/errors/borrowck/rust-bir-builder-pattern.h

```diff
@@ -82,7 +82,9 @@ class PatternBindingBuilder : protected AbstractBuilder,
 
     if (init.has_value ())
       {
-	push_assignment (translated, init.value ());
+	push_assignment (
+	  translated, init.value (),
```

Comment:
Maybe, it would make more sense here to pass the location to the visit identifier method.